### DISCUSSION TITLE
Using Search box to find all files in vhosts with the .wpress extension

### DISCUSSION THREAD

**Discussion by FutureX55** [iterate-ch/cyberduck#15627] [Q&A]
I  typed in .wpress in the search box in the vhosts folder and it ran and ran for an hour or so and never found anything.  am I doing this wrong?  I'm using All-In-One migration plugin I want to find all the AIO archives to delete them and though this would be an easy way.

**Comment by tatze96** [comment]
Cyberduck will search in different depths of the file hierarchy depending on how you use the search bar.

If you type in the search string without confirming it by pressing the Return key, Cyberduck will only search amongst the currently shown files. 

Suppose you confirm the search string by pressing the Return key. In that case, Cyberduck will present a prompt asking you whether you want to search recursively and will search through all items within the currently opened folder upon confirmation of the prompt.

Which of the two search modes did you use?
